build/workbox.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs')
+const swBuild = require('workbox-build')
+const config = require('./workbox.config.json')
+
+const updateUrl = (manifestEntries) => manifestEntries.map((entry) => {
+  if (entry.url.startsWith('_gh_pages/')) {
+    entry.url = entry.url.replace(/_gh_pages\//g, '')
+  }
+  return entry
+})
+
+const workboxSWSrcPath = require.resolve('workbox-sw')
```

Comment:
Yup, everything should be in `_gh_pages` folder. That is what we commit to the gh-pages branch and thus the live site.